docs/src/routes/icon/demo.md

```diff
@@ -0,0 +1,115 @@
+# Icon
+Semantic vector graphics.
+
+## Examples
+
+<Example
+  id="icon-demo-basic"
+  title="Basic"
+  demoComponent="{Basic}"
+  demoCode="{BasicCode}">
+  <p slot="description">
+   Svant offers a list of icons that you can import from <code>svant/icons</code>, there are components for icons with different themes, you can use <code>spin</code> to animate the icon, and <code>rotate</code> to rotate it. </p>
```

Comment:
I noticed yesterday that instead of using <code> tags, we can actually just use backticks in the markdown files HTML and it will get parsed correctly by the preprocessor 😎